Makefile

```
@@ -21,6 +32,7 @@ API_SUPERVISOR_NAME   ?= oullin-sup
 # --- Phony Targets ---
 # Ensures these targets run even if files with the same name exist.
 .PHONY: fresh build-local build run format watch clean clean-extractor build-test sup-api-status sup-api-restart
+.PHONY: ufw-setup ufw-status
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)